src/Microsoft.DncEng.SecretManager/SecretTypes/AzureStorageAccountSasUri.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DncEng.CommandLineLib;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes;
+
+[Name("azure-storage-account-sas-uri")]
+public class AzureStorageAccountSasUri : SecretType<AzureStorageAccountSasUri.Parameters>
```

Comment:
This class doesn't seem to be referenced. What am I missing❔

Reply:
My understanding is that `azure-storage-account-sas-uri` would be set as a type in the KeyVault secret metadata and SecretManager would then use this class through some reflection.

Reply:
That being said, I am still learning about SecretManager